### PR TITLE
Liberate from go/imports to fix #7

### LINKED ISSUE #7: panic-ing on generate if some dependencies have non-gomod module.
Masterminds/squirrel has go.mod, but lann/builder isn't.
goen fails to parse deps like below.

```
/usr/src/goen/example # cat entity.go
package example

import (
        "time"

        "github.com/Masterminds/squirrel"
        uuid "github.com/satori/go.uuid"
)

var _ squirrel.Sqlizer = squirrel.Eq{}

//go:generate goen -o goen.go
type Blog struct {
        BlogID uuid.UUID `goen:"" table:"blogs" primary_key:""`

        Name string

        Author string

        Posts []*Post `foreign_key:"blog_id"`
}

type Post struct {
        Timestamp

        BlogID uuid.UUID `goen:"" table:"posts"`

        PostID int `primary_key:",omitempty"`

        Title string

        Content string

        // "order" is a sql keyword
        Order int

        Blog *Blog `foreign_key:"blog_id"`
}

type Timestamp struct {
        CreatedAt time.Time
        UpdatedAt time.Time
        DeletedAt *time.Time
}
```

```
/usr/src/goen/example # go generate .
2019/11/13 06:40:58 generator.go:143: analyzing package "example"
2019/11/13 06:40:58 generator.go:214: analyzing file "postgres_test.go"
2019/11/13 06:40:58 generator.go:214: analyzing file "sqlite3_test.go"
2019/11/13 06:40:58 generator.go:214: analyzing file "entity.go"
2019/11/13 06:40:58 generator.go:240: analyzing struct type "Blog"
2019/11/13 06:40:58 generator.go:281: analyzing struct field Blog.BlogID as column
panic: goen: unable to assume package name "github.com/Masterminds/squirrel": type-checking package "github.com/Masterminds/squirrel" failed (/go/pkg/mod/github.com/!masterminds/squirrel@v1.1.0/case.go:7:2: could not import github.com/lann/builder (type-checking package "github.com/lann/builder" failed (/go/pkg/mod/github.com/lann/builder@v0.0.0-20180802200727-47ae307949d0/builder.go:5:2: could not import github.com/lann/ps (cannot find package "github.com/lann/ps" in any of:
        /usr/local/go/src/github.com/lann/ps (from $GOROOT)
        /go/src/github.com/lann/ps (from $GOPATH)))))

goroutine 1 [running]:
github.com/kamichidu/goen/internal/asts.AssumePkgName(0xc0003d95c1, 0x1f, 0xa4b12a, 0x1)
        /usr/src/goen/internal/asts/asts.go:107 +0x153
github.com/kamichidu/goen/internal/asts.FindPkgPath(0xc000178300, 0xc0003c7f1c, 0x4, 0x9b6cb, 0x9b6cc)
        /usr/src/goen/internal/asts/asts.go:92 +0x164
github.com/kamichidu/goen/internal.(*aType).Kind.func1(0xb31b60, 0xc0003e3d40, 0x203000, 0x203000)
        /usr/src/goen/internal/structs_a.go:199 +0x2b7
github.com/kamichidu/goen/internal/asts.VisitorFunc.Visit(0xc000372100, 0xb31b60, 0xc0003e3d40, 0x8, 0x40d109)
        /usr/src/goen/internal/asts/asts.go:14 +0x3a
go/ast.Walk(0xb2dc80, 0xc000372100, 0xb31b60, 0xc0003e3d40)
        /usr/local/go/src/go/ast/walk.go:52 +0x66
github.com/kamichidu/goen/internal.(*aType).Kind(0xc0003720e0, 0xc00026b5b0)
        /usr/src/goen/internal/structs_a.go:220 +0xca
github.com/kamichidu/goen/generator.(*Generator).safePkgImport(0xc00026bef8, 0xb3de00, 0xc0003720e0, 0x7, 0x2, 0x3, 0x4)
        /usr/src/goen/generator/generator.go:250 +0x74
github.com/kamichidu/goen/generator.(*Generator).walkStruct(0xc000c4def8, 0xb376c0, 0xc000381be0, 0x1, 0x1)
        /usr/src/goen/generator/generator.go:287 +0x410
github.com/kamichidu/goen/generator.(*Generator).walkFile(0xc000c4def8, 0xc0004bb410, 0xc000178300, 0x1, 0x1)
        /usr/src/goen/generator/generator.go:241 +0x258
github.com/kamichidu/goen/generator.(*Generator).walkPkg(0xc000c4def8, 0xc0004bb410, 0xc00026bac8, 0x1)
        /usr/src/goen/generator/generator.go:216 +0x1e0
github.com/kamichidu/goen/generator.(*Generator).ParseDir(0xc000c4def8, 0x1, 0x1)
        /usr/src/goen/generator/generator.go:151 +0x1db
main.run(0xb2d6a0, 0xc000098000, 0xb2d6e0, 0xc000098008, 0xb2d6e0, 0xc000098010, 0xc000084150, 0x3, 0x3, 0x0)
        /usr/src/goen/cmd/goen/main.go:172 +0x73f
main.main()
        /usr/src/goen/cmd/goen/main.go:208 +0x8a
entity.go:13: running "goen": exit status 2
```

### PR DESCRIPTION
**Pull request by lambdalisue**
Just use `go/build` which can handle `GO111MODULE=on` correctly.

At least this changes fixed issues on my environment.

### CODE REVIEW

File: internal/asts/asts.go [hunk 1]
```diff
@@ -102,11 +102,11 @@ func FindPkgPath(file *ast.File, pkgName string) string {
 }
 
 func AssumePkgName(pkgPath string) string {
-	pkg, err := SrcImporter.Import(pkgPath)
+	bpkg, err := bImport(pkgPath, ".", 0)
 	if err != nil {
-		panic(fmt.Sprintf("goen: unable to assume package name %q: %s", pkgPath, err))
+		panic(err)
```
Comment (kamichidu) [inline internal/asts/asts.go]: unexpected changes?

File: internal/asts/parse.go [hunk 2]
```diff
@@ -3,19 +3,13 @@ package asts
...
 
 func ParsePkgPath(pkgPath string) (*ast.Package, error) {
-	// resolve pkgPath as a vendor if possible
-	tpkg, err := SrcImporter.Import(pkgPath)
-	if err != nil {
-		return nil, err
-	}
 	// resolve directory of pkg path
-	bpkg, err := bImport(tpkg.Path(), ".", build.FindOnly)
+	bpkg, err := bImport(pkgPath, ".", 0)
```
Comment (kamichidu) [inline internal/asts/parse.go]: why not `build.FindOnly` ?

File: internal/asts/asts.go (hunk 1 above)
Comment (lambdalisue) [reply internal/asts/asts.go]: oops. sorry. I'll fix it 👍

File: internal/asts/parse.go (hunk 2 above)
Comment (lambdalisue) [reply internal/asts/parse.go]: To get a correct name in `bpkg.Name`. If you use `build.FindOnly` then `bpkg.Name` is an empty string.

Comment (kamichidu) [reply internal/asts/parse.go]: you're great!

**Pull request merged**